shyftDb/postgres_setup_test/create_tables_test.psql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE IF NOT EXISTS blocks (
+  hash text primary key,
+  coinbase text,
+  gasUsed numeric,
+  gasLimit numeric,
+  txCount numeric,
+  uncleCount numeric,
+  age timestamp,
+  parentHash text,
+  uncleHash text,
+  difficulty bigint,
+  size text,
+  nonce numeric,
+  rewards numeric,
+  number bigint
+);
+
+CREATE TABLE IF NOT EXISTS txs (
+  txHash text,
+  to_addr text,
+  from_addr text,
+  blockhash text references blocks(hash),
+  blocknumber text,
+  amount numeric,
+  gasprice numeric,
+  gas numeric,
+  gasLimit numeric,
+  txFee numeric,
+  nonce numeric,
+  txStatus text,
+  isContract bool,
+  age timestamp,
+  data bytea
+);
+
+CREATE TABLE IF NOT EXISTS accounts (
+  addr text primary key unique,
+  balance numeric,
+  txCountAccount numeric
+);
```

Comment:
Can we not use the other .psql files to do this? That way we don't maintain two sets of identical code?